Sources/NIOHTTP2/HTTP2Error.swift

```diff
@@ -184,12 +184,22 @@ public enum NIOHTTP2Errors {
 
     /// A :status header was received with an invalid value.
     public struct InvalidStatusValue: NIOHTTP2Error {
-        var value: String
+        public var value: String
 
         public init(_ value: String) {
             self.value = value
         }
     }
+
+    /// A priority update was received that would create a PRIORITY cycle.
+    public struct PriorityCycle: NIOHTTP2Error {
+        /// The affected stream ID.
+        public var streamID: HTTP2StreamID
```

Comment:
should this be a let ?

Reply:
No: there is no reason to force fields on structures to be immutable if there is no invariant being broken by their mutation.